manifests/c/COAA/PlanePlotter/6.6.5.3/COAA.PlanePlotter.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: COAA.PlanePlotter
+PackageVersion: 6.6.5.3
+Installers:
```

I meant for this to be part of the previous review, but apparently forgot to hit "Start a review". Apologies.  

```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: MeinbergGlobal.NTP
Installers:
```

---

This thread was marked as resolved, however it still needs to be added.

@blackzafiqz